checkstyle.xml

```diff
@@ -82,6 +82,11 @@
         <module name="ArrayTypeStyle"/>
         <module name="MissingSwitchDefault"/>
         <module name="FallThrough"/>
+        <module name="SimplifyBooleanExpression"/>
+        <module name="SimplifyBooleanReturn"/>
+        <module name="StringLiteralEquality"/>
+        <module name="UnnecessaryParentheses"/>
+        <module name="ExplicitInitializationCheck"/>
```

Comment:
This was deliberately left out so we can be explicit with initializations. It's easy to forget that booleans default to `false`, for example.